lib/ldclient-rb/events.rb

```diff
@@ -481,12 +481,14 @@ def make_output_events(events, summary)
           key: event.key,
           value: event.value,
         }
+
         out[:default] = event.default unless event.default.nil?
         out[:variation] = event.variation unless event.variation.nil?
         out[:version] = event.version unless event.version.nil?
         out[:prereqOf] = event.prereq_of unless event.prereq_of.nil?
-        out[:context] = @context_filter.filter(event.context)
+        out[:context] = @context_filter.filter(event.context, true)
```

Comment:
I'm not so sure as with Go so I'll defer to you, but I think it'd be clearer to have another method that takes the boolean param (`filter_redact_anonymous`) or something? That way most callers don't change.

Reply:
I went ahead and introduced it to keep it consistent. I'll do this in the other implementations when possible.